components/BottomNavigation/tests/unit/BottomNavigationItemViewTests.m

```diff
@@ -127,4 +132,40 @@ - (void)testSetSelectedItemTintColorUpdatesInkColor {
   XCTAssertNotEqualObjects(item2.inkView.inkColor, item2DefaultInkColor);
 }
 
+- (void)testBadgeCenterIsCorrectWithoutRTL {
+  // Given
+  MDCBottomNavigationItemView *itemView = [[MDCBottomNavigationItemView alloc] init];
+  itemView.iconImageView.frame = CGRectMake(8, 8, 24, 24);
+
+  // When
+  CGPoint badgePoint = [itemView badgeCenterFrom:CGRectStandardize(itemView.iconImageView.frame)
+                                           isRTL:NO];
+
+  // Then
+  CGRect iconFrame = itemView.iconImageView.frame;
+  CGPoint expectPoint =
+      CGPointMake(CGRectGetMaxX(iconFrame),
+                  CGRectGetMinY(iconFrame) + kMDCBottomNavigationItemViewBadgeYOffset);
+  XCTAssertEqualWithAccuracy(badgePoint.x, expectPoint.x, 0.001);
+  XCTAssertEqualWithAccuracy(badgePoint.y, expectPoint.y, 0.001);
+}
+
+- (void)testBadgeCenterIsCorrectWithRTL {
+  // Given
+  MDCBottomNavigationItemView *itemView = [[MDCBottomNavigationItemView alloc] init];
+  itemView.iconImageView.frame = CGRectMake(8, 8, 24, 24);
+
+  // When
+  CGPoint badgePoint = [itemView badgeCenterFrom:CGRectStandardize(itemView.iconImageView.frame)
+                                           isRTL:YES];
+
+  // Then
+  CGRect iconFrame = itemView.iconImageView.frame;
+  CGPoint expectPoint =
+      CGPointMake(CGRectGetMinX(iconFrame),
+                  CGRectGetMinY(iconFrame) + kMDCBottomNavigationItemViewBadgeYOffset);
+  XCTAssertEqualWithAccuracy(badgePoint.x, expectPoint.x, 0.001);
+  XCTAssertEqualWithAccuracy(badgePoint.y, expectPoint.y, 0.001);
+}
+
```

Comment:
Looks like you removed the test for the layout to confirm that the badge "switches sides" for RTL. Was there a reason you removed it instead of rewriting it to just check which side the badge is on?

Reply:
Yeah I think the RTL thing is out of scope.